src/Symfony/Bridge/Twig/Extension/CountryExtension.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bridge\Twig\Extension;
+
+use Symfony\Component\Intl\Intl;
+use Twig\Extension\AbstractExtension;
+use Twig\TwigFilter;
+
+/**
+ * Translate country code in the ISO 3166-1 alpha 2 pattern to country name.
+ *
+ * @author Rafael Mello <merorafael@gmail.com>
+ */
+class CountryExtension extends AbstractExtension
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function getFilters()
+    {
+        return array(
+            new TwigFilter('country', array($this, 'getCountryName')),
+        );
+    }
+
+    /**
+     * Return the country name using the Locale class.
+     *
+     * @param string      $isoCode Country ISO 3166-1 alpha 2 code
+     * @param null|string $locale  Locale code
+     *
+     * @return null|string Country name
+     */
+    public function getCountryName($isoCode, $locale = null)
+    {
+        if (null === $isoCode) {
+            return;
+        }
+        if ($locale) {
+            \Locale::setDefault($locale);
+        }
+
+        return Intl::getRegionBundle()->getCountryName($isoCode);
```

Comment:
What about using [`Locale::getDisplayRegion()`](http://php.net/manual/en/locale.getdisplayregion.php) instead?
This would remove the dependency on the Intl component.

Reply:
If a dependency to the Intl-extension is okay. Shouldn't the used dependency, Intl-extension or Intl-component, be added to the suggest-section in composer?

Reply:
@SenseException I added intl-extension as suggested package to use this feature.

Reply:
@nicolas-grekas any position to my bellow comment?